connect/cli/plugins/project/report/templates/bootstrap/project_dir/pyproject.toml.j2

```diff
@@ -0,0 +1,54 @@
+[tool.poetry]
+name = "{{ project_slug|replace("_", "-") }}"
+version = "{{ version }}"
+description = "{{ description }}"
+authors = ["{{ author }}"]
+license = "{{ license }}"
+packages = [
+    { include = "{{ package_name }}" }
+]
+readme = "./README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.8,<4"
+connect-openapi-client = "^22"
+openpyxl = "^3.0.5"
+
+[tool.poetry.dev-dependencies]
+connect-cli = "^22"
```

Comment:
Please set this using a template variable. you can add to the rendering context this variable with the value returned by the `get_version` funcion.